src/addRemoveItems.js

```diff
@@ -0,0 +1,113 @@
+class Task {
+  constructor(description, completed, index) {
+    this.description = description;
+    this.completed = completed;
+    this.index = index;
+  }
+}
+
+let tasks = [];
+const element = document.querySelector('.wrapper');
+const input = document.getElementById('input');
+const complete = document.querySelector('.complete');
+
+export function saveList(task) {
+  localStorage.setItem('myList', JSON.stringify(task));
+  window.location.reload();
+}
+
+export function makeList() {
+  let listItem = '';
+  tasks.forEach((task, index) => {
+    task.index = index + 1;
+    listItem += `<div class="list-container">
+      <div class="list">
+        <input class="check-box" type="checkbox" name="" ${task.completed ? 'checked' : ''}>
+        <input type="text" class ="description ${task.completed ? 'line-through' : ''}" value="${task.description}">
+      </div>
+      <i class="fas fa-ellipsis-v"></i>
+      </div>`;
+    element.innerHTML = listItem;
+  });
+}
+
+export function checkStorage() {
+  if (localStorage.getItem('myList')) {
+    tasks = JSON.parse(localStorage.getItem('myList') || '[]');
+    makeList();
+  }
+}
+
+export function addNewTask() {
+  input.addEventListener('keyup', (e) => {
+    if (e.key === 'Enter' && input.value !== '') {
+      const newTask = new Task(input.value, false, tasks.length + 1);
+      tasks.push(newTask);
+      saveList(tasks);
+      input.value = '';
+    }
+  });
+}
+
+function editAndSave(element) {
+  document.querySelectorAll('.description').forEach((selectedInput, inputIndex) => {
+    selectedInput.addEventListener('keyup', (e) => {
+      tasks.forEach((task) => {
+        if (e.key === 'Enter' && task.index === inputIndex + 1) {
+          task.description = selectedInput.value;
+          element.classList.remove('fa-trash-alt');
+          element.classList.add('fa-ellipsis-v');
+          element.parentElement.style.background = '#fff';
+          element.parentElement.children[0].children[1].style.background = 'none';
+          saveList(tasks);
+        }
+      });
+    });
+  });
+}
```

Comment:
[OPTIONAL]

- [ ] You might want to consider improving the UX for the edit feature. I was initially lost when testing the feature out. A user might not be able to quickly figure out how to edit the task. But all together, this is an awesome project. Great job.